javascript/javascript1/week1/Housey pricey/price_estimator.js

```diff
@@ -0,0 +1,39 @@
+const peterHouseWidthInM2 = 8;
+const peterHouseDepthInM2 = 10;
+const peterHouseHeightInM2 = 10;
+const peterHouseVolumeInMeters =
+  peterHouseDepthInM2 * peterHouseHeightInM2 * peterHouseWidthInM2;
+const peterGardenSizeInM2 = 100;
+
+const peterHouseActualPrice = 2500000;
+
+const peterHouseExpectedPrice =
+  peterHouseVolumeInMeters * 2.5 * 1000 + peterGardenSizeInM2 * 300;
+
+if (peterHouseActualPrice < peterHouseExpectedPrice) {
+  console.log("Peter house price " + peterHouseExpectedPrice + " is too much");
+} else {
+  console.log(
+    "Peter house price " + peterHouseExpectedPrice + " is too little"
+  );
+}
+
+const juliaHouseWidthInM2 = 5;
+const juliaHouseDepthInM2 = 11;
+const juliaHouseHeightInM2 = 8;
+const juliaHouseVolumeInMeters =
+  juliaHouseDepthInM2 * juliaHouseHeightInM2 * juliaHouseWidthInM2;
+const juliaGardenSizeInM2 = 70;
+
+const juliaHouseActualPrice = 1000000;
+
+const juliaHouseExpectedPrice =
+  juliaHouseVolumeInMeters * 2.5 * 1000 + juliaGardenSizeInM2 * 300;
+
+if (juliaHouseActualPrice < juliaHouseExpectedPrice) {
```

Comment:
Same thing as above

Reply:
Added both actual and estimated price in the code